### PR TITLE
Add telnet server switch to enable or disable.

### PR DESCRIPTION
**Pull request by QilongZhang**
#100

### CODE REVIEW

File: sofa-ark-parent/core/common/src/main/java/com/alipay/sofa/ark/common/util/EnvironmentUtils.java [hunk 1]
```diff
@@ -35,6 +35,13 @@ public static String getProperty(String key) {
         return properties.getProperty(key);
     }
 
+    public static String getProperty(String key, String defaultValue) {
+        if (properties.getProperty(key) == null) {
+            return System.getProperty(key, defaultValue);
+        }
+        return properties.getProperty(key);
```
[the remark this answers is not in the document]
Comment (QilongZhang) [reply sofa-ark-parent/core/common/src/main/java/com/alipay/sofa/ark/common/util/EnvironmentUtils.java]: done.

File: sofa-ark-parent/core/spi/src/main/java/com/alipay/sofa/ark/spi/constant/Constants.java [hunk 2]
```diff
@@ -68,6 +68,7 @@
     /**
      * Telnet Server
      */
+    public final static String TELNET_SERVER_ENABLE                  = "telnet.server.enable";
```
[the remark this answers is not in the document]
Comment (QilongZhang) [reply sofa-ark-parent/core/spi/src/main/java/com/alipay/sofa/ark/spi/constant/Constants.java]: done.

File: sofa-ark-parent/core/spi/src/main/java/com/alipay/sofa/ark/spi/constant/Constants.java [hunk 3]
```diff
@@ -68,6 +68,7 @@
     /**
      * Telnet Server
      */
+    public final static String TELNET_SERVER_ENABLE                  = "sofa.ark.telnet.server.enable";
     public final static String TELNET_PORT_ATTRIBUTE                 = "sofa.ark.telnet";
```
Comment (ujjboy) [inline sofa-ark-parent/core/spi/src/main/java/com/alipay/sofa/ark/spi/constant/Constants.java]: How about "sofa.ark.telnet.server.port" ?  Is there a compatibility problem?

Comment (QilongZhang) [reply sofa-ark-parent/core/spi/src/main/java/com/alipay/sofa/ark/spi/constant/Constants.java]: yes. I plan to fix in next major version.

**Pull request merged**